source/codegen/metadata/nirfmxspecan/config_addon.py

```diff
@@ -0,0 +1,6 @@
+# We need to maintain the version here since it needs to be updated by the build process on GitHub
+# TODO
+config_additional_config = {
+    'module_version': '1.3.3.dev0',
```

Comment:
I'm not sure what this needs to be - is this the version of grpc-device?

Reply:
I think this is leftover from nimi-python.